### PR TITLE
[SiStrip DQM] Initial PR for the sistrip control view dqm plots [backport 92X #19204]

### PR DESCRIPTION
**Pull request by jandrea**
Sistrip control view dqm as described in the original PR #19204  
Backport on behalf @gouskos

### CODE REVIEW

**Comment by boudoul** [comment]
Hi @dmitrijus , the PR has been forgotten to be backported, therefore we are catching up , could you pease review it ?  this is equivalent to  #19204   Thank you

**Comment by dmitrijus** [comment]
+1

File: DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc [hunk 1]
```diff
@@ -34,6 +34,7 @@ SiStripMonitorTrack::SiStripMonitorTrack(const edm::ParameterSet& conf):
   Mod_On_        = conf.getParameter<bool>("Mod_On");
   Trend_On_      = conf.getParameter<bool>("Trend_On");
   TkHistoMap_On_ = conf.getParameter<bool>("TkHistoMap_On");
+  clchCMoriginTkHmap_On_ = conf.getParameter<bool>("clchCMoriginTkHmap_On");
```
Comment (dmitrijus) [inline DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc]: this line is extra and not in the 94x pull request
is that intentional?

Comment (dmitrijus) [reply DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc]: it was added in this pull request https://github.com/cms-sw/cmssw/pull/19426
maybe it should be backported too?

Comment (gouskos) [reply DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc]: Hi @dmitrijus @jandrea  this line was not added by us. I am not sure what would be the best

**Comment by davidlange6** [comment]
it prevents you from using an uninitialized variable (presumably a bug in your code). Probably worth including,

**Comment by fioriNTU** [comment]
I agree with @davidlange6 , that line add a protection. Please include it in the PR

**Comment by davidlange6** [comment]
CMSSW_9_2_X developement work is completed. Closing remaining PRs

**Pull request closed without merging**